### PR TITLE
DOCSP-40745  Version support disclaimer at the top-level database tools page

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Recommend users use the latest release version for database tools

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/database-tools/DOCSP-40745/

## JIRA
https://jira.mongodb.org/browse/DOCSP-40745?filter=-1

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66a3c6ea26eba795526c1284

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -68,6 +68,11 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the 
+   {+dbtools+}.
+
```
Comment (ianf-mongodb) [inline source/index.txt:75]: Can we add a small code snippet here showing users how to check their current tool version?

I believe that checking any of the tool binaries should indicate the current version of the tools installed.

So something like:

```
To check your current version of the {+dbtools+}, run the following command on your terminal:

.. code-block:: shell

   mongodump --version

```

**Review by ianf-mongodb — COMMENTED** [review]
Hi @jocelyn-mendez1, I left a suggestion for the edit. Thanks!

File: source/index.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/index.txt:81]: Thanks Ian, it has been updated!

**Comment by jocelyn-mendez1** [comment]
Thanks @ianf-mongodb for the feedback, let me know of any other possible updates!

File: source/index.txt [hunk 2]
```diff
@@ -68,6 +68,17 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the 
+   {+dbtools+}. To check your current version of the {+dbtools+}, run the
```
Comment (ianf-mongodb) [inline source/index.txt:74]: **Nit**

Small nit, Im actually not sure which one sounds better feel free to disregard. 

```suggestion
   {+dbtools+}. To check the version of the {+dbtools+}, run the 
```

**Review by ianf-mongodb — APPROVED** [review]
Thanks @jocelyn-mendez1 LGTM

**Review by rohshar — APPROVED** [review]

File: source/index.txt [hunk 3]
```diff
@@ -68,6 +68,17 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the
```
Comment (dsahni) [inline source/index.txt:73]: Can we also tell users where to find what the latest version is? 

We could link to the download center:
https://www.mongodb.com/try/download/database-tools

Comment (jocelyn-mendez1) [reply source/index.txt:73]: Definitely, its been added!

**Comment by jocelyn-mendez1** [comment]
Thanks for the feedback @dsahni, please let me know of any other possible updates!

**Review by dsahni — APPROVED** [review]
thanks @jocelyn-mendez1, lgtm

**Pull request merged**